azure-pipelines.yml

```diff
@@ -102,8 +102,47 @@ jobs:
                $(Slack.URL)
         condition: and(failed(), eq(variables['Build.SourceBranchName'], 'master'))
 
+  - job: Windows_signing
+    # Signing is a separate job so that we can make sure that we only sign on releases.
+    # Since the release check is run on Linux, we do not have access to that information
+    # in the regular Windows step.
+    dependsOn: [ "Windows", "Linux" ]
+    pool:
+      name: 'windows-pool'
+    condition: and(succeeded(), eq(dependencies.Linux.outputs['release.has_released'], 'true'))
+    variables:
+      unsigned-installer: $[ dependencies.Windows.outputs['publish.artifact-unsigned-windows-installer'] ]
+    steps:
+      - checkout: self
+        persistCredentials: true
+      - bash: |
+          set -euo pipefail
+          echo "##vso[task.setvariable variable=release_tag]$(cat VERSION)"
+        name: release
+      - task: DownloadPipelineArtifact@0
+        inputs:
+          artifactName: $(unsigned-installer)
+          targetPath: $(Build.StagingDirectory)/
+      - bash: |
+          set -euo pipefail
+          INSTALLER=daml-sdk-$(release_tag)-windows.exe
+          mv "$(Build.StagingDirectory)/$(unsigned-installer)" "$(Build.StagingDirectory)/$INSTALLER"
+          chmod +x "$(Build.StagingDirectory)/$INSTALLER"
+          echo "$SIGNING_KEY" | base64 -d > signing_key.pfx
+          MSYS_NO_PATHCONV=1 signtool.exe sign '/f' signing_key.pfx '/fd' sha256 '/tr' "http://timestamp.digicert.com" '/v' "$(Build.StagingDirectory)/$INSTALLER"
+          rm signing_key.pfx
+          echo "##vso[task.setvariable variable=artifact-windows-installer;isOutput=true]$INSTALLER"
+          echo "##vso[task.setvariable variable=has_released;isOutput=true]true"
```

Comment:
I don't see this variable being used. Should it be added to the checks of the `release` job?

Reply:
I’ve just removed it. Given that we set it unconditionally there doesn’t seem to be a reason to have it.